localization/english/imperia_money_transfer_l_english.yml

```diff
@@ -0,0 +1,12 @@
+﻿l_english:
+ BUDGET_TAB_MONEY_TRANSFER: "Transfer"
+ TRANSFER_MONEY_SEND_MONEY: "Select Recipient"
+ MONEY_TRANSFER_REALLY_SEND_MONEY: "Really Send Money to [GetPlayer.MakeScope.Var('money_transfer_target_country').GetCountry.GetName]?"
+ CANT_SEND_MONEY_EXPLANATION: "[GetPlayer.GetCustom('imperia_cant_send_money_explanation')]"
+ CANT_SEND_MONEY_EXPLANATION_DESC: "You #r #b CANNOT#!#! send that amount of money to this country, because of:[GetPlayer.GetCustom('imperia_cant_send_money_too_little_cash_on_hand')][GetPlayer.GetCustom('imperia_cant_send_money_too_little_space_in_target')][GetPlayer.GetCustom('imperia_cant_send_money_must_send_money')]"
+ CAN_SEND_MONEY_EXPLANATION_DESC: "[GetPlayer.GetNameNoFlag] loses @money![Negate_CFixedPoint(GetPlayer.MakeScope.Var('imperia_money_transfer_amount').GetValue)|K+]\n[GetPlayer.MakeScope.Var('money_transfer_target_country').GetCountry.GetName] gets @money![GetPlayer.MakeScope.Var('imperia_money_transfer_amount').GetValue|K+=]"
+ CANT_SEND_MONEY_TOO_LITTLE_CASH_ON_HAND: "\n#r Too little in your reserve and/or max credit limit.#!"
```

Comment:
"The amount exceeds your gold reserve plus credit limit. You would go beyond bankrupt."